bitnami/keycloak/templates/secrets.yaml

```
@@ -25,7 +25,7 @@ data:
   management-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{- end }}
   {{- if and (not .Values.postgresql.enabled) (not .Values.externalDatabase.existingSecret) }}
-  postgresql-password: {{ .Values.externalDatabase.password | b64enc | quote }}
+  database-password: {{ .Values.externalDatabase.password | b64enc | quote }}
```

This breaks backwards compatibility (forcing you to bump the major) but I think it's something we can workaround. Check my suggestion:
```suggestion
  {{/*
  We use a different key for PostgreSQL for backwards compatibility, see https://github.com/bitnami/charts/pull/8108
  TO BE DEPRECATED in 6.0.0
  */}}
  {{- if eq .Values.externalDatabase.vendor "postgres" }}
  postgresql-password: {{ .Values.externalDatabase.password | b64enc | quote }}
  {{- else }}
  database-password: {{ .Values.externalDatabase.password | b64enc | quote }}
  {{- end }}
```